src/EventHandlers/Pool.ts

```diff
@@ -57,16 +57,24 @@ Pool.Fees.handlerWithLoader({
 
     // load the token entities
     const token0Instance = await context.Token.get(
-      currentLiquidityPool.token0_id
+      currentLiquidityPool.token0_id.toLowerCase()
     );
     const token1Instance = await context.Token.get(
-      currentLiquidityPool.token1_id
+      currentLiquidityPool.token1_id.toLowerCase()
```

Comment:
Same comment. I've seen the hourly snapshots are still checksummed addresses. Having some checksummed and some not will be quite chaotic in end queries and dashboard intelligence